src/main/java/seedu/address/model/todolist/Priority.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.model.todolist;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents the priority of a todoList event.
+ * Guarantees: immutable; is valid as declared in {@link #isValid(String)}
+ * Setting Rule: user can set 1 for High; 2 for Medium; 3 for Low
+ */
+
+public class Priority {
+
+    public static final String DEFAULT_PRIORITY = "1"; // default field name
+    public static final String MESSAGE_CONSTRAINTS =
+            "The valid user input for priority is p/[priority]."
+                    + "While [priority] is a integer from 1 to 3,"
```

Comment:
probably more user friendly if the user needs to key-in words for the priority rather than number. but this is a minor issue and just a suggestion